MoreCombineVoiceLines.lua

```diff
@@ -0,0 +1,616 @@
+Schema.voices.Add("comb", "STAB", "Stab", "npc/comb_soldier/vo/stab.wav")
+Schema.voices.Add("comb", "STABILZATION TEAM AT SECTOR CONTROL", "Stabilzation team at sector control", "npc/comb_soldier/vo/stabilizationteamhassector.wav")
+Schema.voices.Add("comb", "STABILZATION TEAM HOLDING AT POSTION", "Stabilzation team holding at position", "npc/comb_soldier/vo/stabilizationteamholding.wav")
+Schema.voices.Add("comb", "STANDING BY", "Standing by", "npc/comb_soldier/vo/standingby].wav")
+Schema.voices.Add("comb", "STAR", "Star", "npc/comb_soldier/vo/star.wav")
+Schema.voices.Add("comb", "STAY ALERT", "Stay alert", "npc/comb_soldier/vo/stayalert.wav")
+Schema.voices.Add("comb", "STAY ALERT REPORT SIGHT LINES", "Stay alert report sight lines" ,"npc/comb_soldier/vo/stayalertreportsightlines.wav")
+Schema.voices.Add("comb", "STINGER", "Stinger", "npc/comb_soldier/vo/stinger.wav")
+Schema.voices.Add("comb", "STORM", "Storm", "npc/comb_soldier/vo/storm.wav")
+Schema.voices.Add("comb", "STRIKER", "Striker", "npc/comb_soldier/vo/striker.wav")
+Schema.voices.Add("comb", "SUNDOWN", "Sundown", "npc/comb_soldier/vo/sundown.wav")
+Schema.voices.Add("comb", "SUPPRESSING", "Suppressing", "npc/comb_soldier/vo/suppressing.wav")
+Schema.voices.Add("comb", "SWARM OUT BREAK IN SECTOR", "Swarm out break in sector", "npc/comb_soldier/vo/swarmoutbreakinsector.wav")
+Schema.voices.Add("comb", "SWEEPER", "Sweeper", "npc/comb_soldier/vo/sweeper.wav")
+Schema.voices.Add("comb", "SWEEPING IN", "Sweeping in", "npc/comb_soldier/vo/sweepingin.wav")
+Schema.voices.Add("comb", "SWIFT", "Swift", "npc/comb_soldier/vo/swift.wav")
+Schema.voices.Add("comb", "SWORD", "Sword", "npc/comb_soldier/vo/sword.wav")
+Schema.voices.Add("comb", "TARGET", "Target", "npc/comb_soldier/vo/target.wav")
+Schema.voices.Add("comb", "TARGET COMPROMISED", "Target compromised move in, move in", "npc/comb_soldier/vo/targetcompromisedmovein.wav")
+Schema.voices.Add("comb", "WARP UP", "Thats it warp up", "npc/comb_soldier/vo/thatsitwrapitup.wav")
+Schema.voices.Add("comb", "UNIFORM", "Uniform", "npc/comb_soldier/vo/thatsitwrapitup.wav")
+Schema.voices.Add("comb", "UNIT IS CLOSING", "Unit is closing", "npc/comb_soldier/vo/unitisclosing.wav")
+Schema.voices.Add("comb", "UNIT IS INBOUND", "Unit is inbound", "npc/comb_soldier/vo/unitisinbound.wav")
+Schema.voices.Add("comb", "VAMP", "Vamp", "npc/comb_soldier/vo/vamp.wav")
+Schema.voices.Add("comb", "VISCON", "Viscon", "npc/comb_soldier/vo/viscon.wav")
+Schema.voices.Add("comb", "VISUAL ON EXOGENS", "Visual on exogens", "npc/comb_soldier/vo/visualonexogens.wav")
+Schema.voices.Add("comb", "WEAPONS OFF SAFE", "Weapons off safe prep for contact")
+Schema.voices.Add("comb", "WE HAVE FREE PARASITES", "We have free parasites", "npc/comb_soldier/vo/wehavefreeparasites.wav")
+Schema.voices.Add("comb", "WINDER", "Winder", "npc/comb_soldier/vo/winder.wav")
+
+Schema.voices.AddClass("comb", function(client)

```

Comment:
Again, the HL2 RP schema already defines the `Combine` voice class. You should use the existing voice class instead of making your own that achieves a similar result anyway.